xml/System.Windows.Media/ImageBrush.xml

```diff
@@ -32,7 +32,7 @@ An ImageBrush can paint shapes, controls, text, and more
  For more information about <xref:System.Windows.Media.ImageBrush> features, see [Painting with Images, Drawings, and Visuals](~/docs/framework/wpf/graphics-multimedia/painting-with-images-drawings-and-visuals.md) overview.  
   
 ## Freezable Features  
- Because the <xref:System.Windows.Media.ImageBrush> class inherits from <xref:System.Windows.Freezable>, <xref:System.Windows.Media.ImageBrush> objects gain several special features, which include the following: they can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread safe. For more information about the features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ Because the <xref:System.Windows.Media.ImageBrush> class inherits from <xref:System.Windows.Freezable>, <xref:System.Windows.Media.ImageBrush> objects gain several special features, which include the following: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread safe. For more information about the features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 Because the <xref:System.Windows.Media.ImageBrush> class inherits from <xref:System.Windows.Freezable>, <xref:System.Windows.Media.ImageBrush> objects gain several special features, which include the following: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread safe. For more information about the features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```